### PR TITLE
[ETCM-661]: adds pottery network for local nomad/docker setup

### PR DESCRIPTION
**Pull request by krzysztofpaliga-iohk**
# Description

For nomad/docker configuration we need a fresh local network, which runs on its own.

# Proposed Solution

A new network can called pottery has been introduced.

### CODE REVIEW

File: src/main/resources/conf/chains/pottery-chain.conf [hunk 1]
```diff
@@ -0,0 +1,160 @@
+{
```
Comment (jvdp) [inline src/main/resources/conf/chains/pottery-chain.conf]: Is there stuff you can use from an existing chain conf perhaps? Then you could just include that and keep only the changed params.

File: src/main/resources/conf/chains/pottery-genesis.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "difficulty": "0x20000",
```
Comment (jvdp) [inline src/main/resources/conf/chains/pottery-genesis.json]: I think a lower difficulty is possible: https://github.com/input-output-hk/mantis/blob/develop/src/rpcTest/resources/privateNetConfig/conf/rpc-test-private-genesis.json#L5

File: src/main/resources/conf/pottery.conf [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+mantis {
+  blockchains {
+    network = "pottery"
+  }
+
+  network {
+    protocol-version = 64
+
+    discovery {
+      discovery-enabled = true
```
Comment (jvdp) [inline src/main/resources/conf/pottery.conf:12]: How does this work? Every node will need to be bootstrapped somewhere, right?

**Review by jvdp — APPROVED** [review]
Not much to say about config files (other than a few specific points in my comments.) Great to have dedicated configuration for a local testnet, though 👍  Also keeps https://github.com/input-output-hk/mantis-automation a bit simpler, I imagine?

File: src/main/resources/conf/pottery.conf (hunk 3 above)
Comment (krzysztofpaliga-iohk) [reply src/main/resources/conf/pottery.conf:12]: As far as I know, every node uses bootstrapped nodes. I had some issues without discovery enabled. I can't recall what it was. : /

File: src/main/resources/conf/chains/pottery-chain.conf (hunk 1 above)
Comment (krzysztofpaliga-iohk) [reply src/main/resources/conf/chains/pottery-chain.conf]: good idea...

File: src/main/resources/conf/chains/pottery-genesis.json (hunk 2 above)
Comment (krzysztofpaliga-iohk) [reply src/main/resources/conf/chains/pottery-genesis.json]: good idea...

**Review by leo-bogastry — APPROVED** [review]

**Pull request merged**